hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -943,9 +943,13 @@ private OmMultipartInfo createMultipartInfo(OmKeyArgs keyArgs,
 
       long currentTime = Time.now();
       Map<Integer, PartKeyInfo> partKeyInfoMap = new HashMap<>();
-      OmMultipartKeyInfo multipartKeyInfo = new OmMultipartKeyInfo(
-          multipartUploadID, currentTime, keyArgs.getType(),
-          keyArgs.getFactor(), partKeyInfoMap);
+      OmMultipartKeyInfo multipartKeyInfo = new OmMultipartKeyInfo.Builder()
```

Comment:
Is this code still used? @bharatviswa504

Reply:
No @arp7. For write requests new OM HA code is used.

Reply:
Should we file a separate jira to delete this code now?